src/partials/footer.html

```diff
@@ -1,14 +1,14 @@
 <footer>
```

Comment:
cały footer powinien być z klasą container, by nie robić oddzielnych marginesów w każdej sekcji strony - klasa container to wszystko przejmie.